src/main/java/org/isf/menu/gui/UserBrowsing.java

```diff
@@ -121,250 +86,239 @@ public void userUpdated(AWTEvent e) {
 	private JComboBox pbox;
 	private ArrayList<User> pUser;
 	private String[] pColumns = {
-			MessageBundle.getMessage("angal.menu.userm"), 
-			MessageBundle.getMessage("angal.menu.groupm"), 
-			MessageBundle.getMessage("angal.menu.descm") };
-	private int[] pColumnWidth = {70, 70, 150 };
+			MessageBundle.getMessage("angal.userbrowser.user.txt"),
+			MessageBundle.getMessage("angal.userbrowser.group.txt"),
+			MessageBundle.getMessage("angal.userbrowser.description.txt") };
+	private int[] pColumnWidth = { 70, 70, 150 };
 	private User user;
-	private DefaultTableModel model ;
+	private DefaultTableModel model;
 	private JTable table;
 	private JScrollPane scrollPane;
-	
+
 	private String pSelection;
-	
+
 	private UserBrowsing myFrame;
 	private UserBrowsingManager manager = Context.getApplicationContext().getBean(UserBrowsingManager.class);
 
 	public UserBrowsing() {
-		
-		setTitle(MessageBundle.getMessage("angal.menu.usersbrowser"));
+
+		setTitle(MessageBundle.getMessage("angal.userbrowser.title"));
 		myFrame = this;
-		
-		//addWindowListener(this);
-		
+
 		setSize(DEFAULT_WIDTH, DEFAULT_HEIGHT);
 		Toolkit kit = Toolkit.getDefaultToolkit();
 		Dimension screensize = kit.getScreenSize();
-		pfrmWidth = screensize.width / 2 + 100;
+		pfrmWidth = (screensize.width / 2) + 100;
 		pfrmHeight = screensize.height / 2;
-		setBounds(screensize.width / 4, screensize.height / 4, pfrmWidth,
-				pfrmHeight);
-		
+		setBounds(screensize.width / 4, screensize.height / 4, pfrmWidth, pfrmHeight);
+
```

Comment:
We should be able to get rid of these sizing still used in many classes and try `pack(); setLocationRelativeTo(null);` combination

Reply:
I will look into this later if that is OK?

Reply:
Used your suggestion; it works although the dialog is now a little smaller in size.

Reply:
Yep, it is the effect of packing, the layout manager expects we play with padding and margins, if we like